config.json

```diff
@@ -57,6 +57,18 @@
         "strings"
       ]
     },
+    {
+      "slug": "isogram",
+      "uuid": "3c1b70fa-aaa8-4323-9aab-4c76da0a89e1",
+      "core": false,
+      "unlocked_by": null,
```

Comment:
@jvarness unlocked_by should have "armstrong-numbers" as the value, unless you want to make it a core exercise.

Reply:
Oh I wasn't sure if you meant the position in the file or if you wanted it to be unlocked by armstrong numbers. I'll change that.

Reply:
Set `"unlocked_by": "armstrong-numbers"`